### ISSUE TITLE
Version mismatch in maven central repo

### ISSUE THREAD

**Issue by higauravkohli** [awslabs/amazon-sqs-java-messaging-lib#200]
Hello, 
I'm trying to download v1.1.3 of the library from maven central, 
https://repo.maven.apache.org/maven2/com/amazonaws/amazon-sqs-java-messaging-lib/1.1.3/
However, there's a version mismatch in the pom.xml file. Although its uploaded under 1.1.3 folder, teh version inside the pom.xml is 2.0.3. Could you please fix this, as this is blocking our development.
![image](https://github.com/awslabs/amazon-sqs-java-messaging-lib/assets/19509106/a6ff867e-5c1f-4d39-8cb0-498d05d23d69)
Best,
Gaurav

**Comment by pzygielo** [comment]
- sounds like duplicate of #174

**Comment by higauravkohli** [comment]
yes, however, I see that #174  was closed without any resolution. Will this issue be fixed?
